admins/pageflow/entry.rb

```diff
@@ -49,7 +49,9 @@ module Pageflow
     end
 
     filter :title
-    filter :account
+    filter :account,
+           if: ->(_) { authorized?(:index, :accounts) },
+           collection: -> { Account.accessible_by(current_ability, :see_in_list) }
```

Comment:
Sorry for being so nitpicky, but in the `Folder` case in line 67 below we are also reusing `:read` which maps to `#resolve`. Why not do the same thing here? Also there is one Hound warning left.